torch/distributions/fishersnedecor.py

```diff
@@ -50,13 +50,13 @@ def expand(self, batch_shape, _instance=None):
     @property
     def mean(self):
         df2 = self.df2.clone()
-        df2[df2 <= 2] = nan
+        df2[(df2 <= 2).byte()] = nan
```

Comment:
Note "masked functions do not accept bool yet"

Reply:
Actually, breakage of this code is very serious, and I think a very strong argument that you have to fix the masked functions first before landing this diff.